ci/github/github_setup_pytest_replay

```diff
@@ -0,0 +1,5 @@
+mkdir ./pytest-replay
+declare -a PYTEST_REPLAY_OPTIONS=()
+PYTEST_REPLAY_OPTIONS+=("--replay-record-dir=./pytest-replay")
+PYTEST_REPLAY_OPTIONS+=("--replay-base-name=Linux-$CONDA_VERSION-Py$PYTHON_VERSION")
+echo "${PYTEST_REPLAY_OPTIONS[@]}"
```

Comment:
Yes, I see that this is run in the `Source Scripts` job but I don't think it's actually used?

Reply:
Its being sourced at in the "Source Scripts" job of the tests.yml

Reply:
This is how they did it in the azure pipeline so I just duplicated mostly what was done within the Github structure, but should we be sourcing scripts like this? Or should we keep this scripting within the action yaml itself?

Reply:
Both of the replay options defined here are identical to those defined as `env.pytest-replay` so those appears to be duplicates.

The only command that appears to be a non-duplicate is the `mkdir` command. I'd be inclined to nuke this setup script and move the `mkdir` command into the workflow itself.